requirements.txt

```diff
@@ -12,3 +12,4 @@ six
 setuptools>=18.0
 pyyaml
 xarray
+statsmodels
```

Comment:
I don't know anything about it. However, it is offered with Anaconda (and PyPI) so it's probably pretty stable. How necessary is it @saisiddu? It looks like you just use if for `empirical_distribution` in one location so probably not a big deal include it.